drivers/spi/spi_sifive.c

```diff
@@ -0,0 +1,319 @@
+/*
+ * Copyright (c) 2018 SiFive Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define SYS_LOG_LEVEL CONFIG_SYS_LOG_SPI_LEVEL
+#include <logging/sys_log.h>
+
+#include "spi_context.h"
+#include "spi_sifive.h"
+
+#include <stdbool.h>
+
+#include <device.h>
+#include <spi.h>
+
+#define MIN(x, y) ((x < y) ? x : y)
+
+/* Structure Declarations */
+
+struct spi_sifive_data {
+	struct spi_context ctx;
+};
+
+struct spi_sifive_cfg {
+	u32_t base;
+	u32_t f_sys;
+};
+
+/* Helper Functions */
+
+int spi_config(struct device *dev, u32_t frequency, u16_t operation)
+{
+	if (SPI_OP_MODE_GET(operation) != SPI_OP_MODE_MASTER)
```

Comment:
We add {} even for one-liners if/for/while/etc... in Zephyr. Please apply that everywhere.
(Note that you are doing it properly already in transceive function)